dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -90,4 +100,5 @@ EXPOSE {{ $port }}
 {{- end }}
 
 WORKDIR {{ $beatHome }}
-ENTRYPOINT ["/usr/local/bin/docker-entrypoint"]
+ENTRYPOINT ["/usr/bin/tini", "--", "/usr/local/bin/docker-entrypoint"]
+CMD ["-environment", "container"]
```

Comment:
@mieciu sorry, one last thing. I have just seen that this is being added here, it was not here before and actually I think that agent doesn't have this option.
```suggestion
```

Reply:
Nice catch!